### PR TITLE
implement goerli FeeVault fix

### PR DESCRIPTION
**Pull request by dapperscene6**
An issue was discovered with the FeeVault implementation, and the vault contract state variable tracking total processed funds has gone out of sync. The PR deploys and upgrades the vaults (SequencerFeeVault, L1FeeVault, BaseFeeVault).

The scripts perform the following:
1. `DeployNewFeeVaultsL2.sol`: Deploy Base’s FeeVault contract implementations as required to implement the fix. This will require a total of **two (2)** contracts per vault (for a total of 6).
   * (Given that the `totalProcessed` incrementation issue has already been triggered on all Base Goerli’s FeeVaults we’ll conduct a custom contract upgrade which will set Base Goerli’s FeeVault’s to match the total ETH fees processed prior to the the `totalProcessed` incrementation issue being introduced before setting the final implementation code.)
2. `FixFeeVaultsL2.sol`: Upgrade the FeeVault contract by going through the first intermediate contract which sets the `totalProcessed` variable to the correct amount, and then upgrades to the final implementation contract code.

### CODE REVIEW

File: goerli/2023-08-24-feevault-fix/.env [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+DEPLOYER=0xf2faee04500bcd6797cb8545f9edd7c933b99835
+
+SEQUENCER_FEEVAULT_IMPL_INT=0xd7C5851c312F9bEdD7eCD9C65B1d1CBe9Be58378
+L1_FEEVAULT_IMPL_INT=0x1699913846798e923E3d5CF05F83ddFd8684e415
+BASE_FEEVAULT_IMPL_INT=0xFB78c2719981Cdae351cE9D93A7cBF4cc92b18b0
+SEQUENCER_FEEVAULT_IMPL_FINAL=0x6e831b81D80466455Bed19B3f15F6bdb20e18150
+L1_FEEVAULT_IMPL_FINAL=0x2760458293cb7a9901a872bD5299dcb92B96848a
+BASE_FEEVAULT_IMPL_FINAL=0x6E3A399D6E6e12c83e15F15beBBF16f69AB46907
+
+# cast call 0x4200000000000000000000000000000000000011 "totalProcessed()(uint256)" --rpc-url https://base-goerli.cbhq.net --block 7607754
```
Comment (anikaraghu) [inline goerli/2023-08-24-feevault-fix/.env:19]: 👍 very helpful comment

File: goerli/2023-08-24-feevault-fix/.env [hunk 2]
```diff
@@ -0,0 +1,27 @@
+OP_COMMIT=90bff30727d05fb3451e2a6a118c1350fd0c62af
```
Comment (anikaraghu) [inline goerli/2023-08-24-feevault-fix/.env:1]: Verified everything in this file

File: goerli/2023-08-24-feevault-fix/Makefile [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+ifndef LEDGER_ACCOUNT
+override LEDGER_ACCOUNT = 0
+endif
+
+##
+# Commands for signers to run to test the nested safe
+##
+.PHONY: sign-op
+sign-op:
```
Comment (anikaraghu) [inline goerli/2023-08-24-feevault-fix/Makefile:20]: nit: maybe just `sign1` and `sign2` as OP is not involved on testnet, but also this is fine if we want to keep consistency

**Review by anikaraghu — APPROVED** [review]
LGTM

**Review by anikaraghu — COMMENTED** [review]
@dapperscene6 can we deduplicate from https://github.com/base-org/contract-deployments/pull/80 ? Maybe close that one and pull in the PR description here?

File: goerli/2023-08-24-feevault-fix/Makefile (hunk 3 above)
Comment (dapperscene6) [reply goerli/2023-08-24-feevault-fix/Makefile:20]: For now I'd like to keep it as op / cb for consistency. What if in the future we call it `sign-op-fake` to denote the command represents Optimism's portion of the multisig but doesn't actually involve Optimism?

**Pull request merged**